site/content/resources/security-checklist.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Security Response Checklist
+layout: page
+---
+
+This document outlines a checklist for Contour security team members (at time of writing, this is the same people as the maintainer team) to step through in the event Contour has a CVE that needs to be mitigated.
+
+## A CVE has been reported, what do I do?
+
+1. User discovers a vulnerability and notifies cncf-contour-maintainers@lists.cncf.io
+1. Contour maintainer team triages the vulnerability with the reporter and decide patch releases (multiple minors could be impacted) as well as downstream distributors.
+1. Create a Security Advisory Draft on github Contour repo https://github.com/projectcontour/contour/security/advisories
```

Comment:
maybe note that development on the CVE fix could be done in a private fork as part of the Security Advisory process? (if needed)

Reply:
yeah, that's the way to go, that makes things much easier to control. I'll add in.